lib/clientStubFromSlash.js

```diff
@@ -6,6 +6,7 @@ var Url = require("url-parse");
 var clientStub_1 = require("./clientStub");
 var PORT = "443";
 function clientStubFromSlashGraphQLEndpoint(graphqlEndpoint, apiKey) {
+    console.warn("This method is deprecated and will be removed in v21.07 release.");
```

Comment:
I assume this is the best practice to deprecate a function in `js`.

Reply:
The ts file `src/clientStubFromSlash.ts` has the `@deprecated` for the function as well.

Reply:
I was wondering if this would pollute user's console logs. `@deprecated` in function should be fine I guess.